### DISCUSSION TITLE
Sunbird collection editor for DIKSHA - Version used and branch

### DISCUSSION THREAD

**Discussion by pushpamtrigyn** [Sunbird-Knowlg/Community#151] [Documentation Issues]
Hi,

Currently DIKSHA application is using the following Sunbird collection editor version :-

"@project-sunbird/sunbird-collection-editor-v9": "5.0.14"

I was able to trace the repository to https://github.com/Sunbird-Knowlg/sunbird-collection-editor. However, for the v9 version and 5.0.14 npm package version, I am not sure from which branch or tag this was built from.

We actually require to fix an issue where the question set editor is not loading on our environments, so require to fork the corresponding repo and make changes to it. Please assist for the same.

CC - @vinukumar-vs

**Comment by pallakartheekreddy** [comment]
@pushpamtrigyn We have created tags for release-5.0.0 branch. Please use those tags for any fixes
`https://github.com/Sunbird-Knowlg/sunbird-collection-editor/releases/tag/release-5.0.0_RC14`
@Krishnaj20 @vinukumar-vs

**Reply by pallakartheekreddy** [reply]
@pushpamtrigyn Hope your query is now resolved. Please mark the thread as answered

**Comment by pallakartheekreddy** [comment]
@pushpamtrigyn As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
@Krishnaj20
